llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -3901,6 +3958,26 @@ SDValue SITargetLowering::lowerPREFETCH(SDValue Op, SelectionDAG &DAG) const {
   return Op;
 }
 
+// Work around DAG legality rules only based on the result type.
+SDValue SITargetLowering::lowerFP_EXTEND(SDValue Op, SelectionDAG &DAG) const {
+  bool IsStrict = Op.getOpcode() == ISD::STRICT_FP_EXTEND;
```

Comment:
tiny tiny nit: put parentheses around the initializer - `=` and `==` on each side of an expression is just a bit confusing at first glance